### ISSUE TITLE
Is there plan to support dll packer and x64 PE file?

### ISSUE THREAD

**Issue by bestofcham** [azerg/PackerPE#3]

**Comment by azerg** [comment]
Yep, when time permits. I will have to build a loader and it will be a 90% of a x64 PE support :)
The trouble is i cant use inline asm in x64 proj... 
Anyway will be working on this first.
